components/compliance-service/migrations/upgrades.go

```diff
@@ -20,27 +20,15 @@ func NewService(pg *pgdb.UpgradesDB, cerealManger *cereal.Manager) *Upgrade {
 
 //PollForUpgradeFlagDayLatest checks for the day latest flag value in upgrade flags
 func (u *Upgrade) PollForUpgradeFlagDayLatest() error {
-
-	status, err := u.storage.GetDayLatestUpgradeFlagValue()
+	flagMap, err := u.storage.GetUpgradeFlags()
 	if err != nil {
-		logrus.Errorf("Unable to get the status of day latest flag")
-		return errors.Wrapf(err, "Unable to get the status of day latest flag")
+		logrus.Errorf("Unable to get the status of upgrade flags")
+		return errors.Wrapf(err, "Unable to get the status of upgrade flags")
 	}
-	err = u.cerealInterface.EnqueueWorkflowDayLatest(status)
+	err = u.cerealInterface.EnqueueWorkflowUpgrade(flagMap[pgdb.DayLatestFlag], flagMap[pgdb.ControlIndexFlag], flagMap[pgdb.CompRunInfoFlag])
```

Comment:
not convinced with this way of triggering the cereal service, lets discuss over the call